pkgs/development/compilers/halide/default.nix

```diff
@@ -30,7 +30,7 @@ llvmPackages.stdenv.mkDerivation rec {
     sed -i "s/-Werror//" src/CMakeLists.txt
```

Comment:
Is this still necessary? Generally newer versions of CLANG add support for new flags.

Reply:
Oh lemme test that too, thx for the review!

Reply:
It worked without it

Reply:
The Halide build doesn't inflict `-Werror` on its users